h2o-py/h2o/h2o.py

```diff
@@ -1969,7 +1969,7 @@ def load_dataset(relative_path):
 
 
 def make_metrics(predicted, actual, domain=None, distribution=None, weights=None, treatment=None, auc_type="NONE",
-                 auuc_type="AUTO", auuc_nbins=-1):
+                 auuc_type="AUTO", auuc_nbins=-1, custom_auuc_thresholds=None):
```

Comment:
These changes enable custom thresholds to calculate metrics and could be in a different PR, but it is necessary for tests. But I can move it.